JustFakeIt/ExpectationExtensions.cs

```diff
@@ -53,7 +56,48 @@ public static bool MatchesActualHttpMethod(this HttpRequestExpectation expected,
 
         public static bool MatchesActualBody(this HttpRequestExpectation expected, string actualBody)
         {
-            return string.IsNullOrEmpty(expected.Body) || actualBody.Equals(expected.Body);
+            return MatchBody(expected.Body, actualBody);
+        }
+
+        public static bool MatchBody(string expectedBody, string actualBody)
+        {
+            return string.IsNullOrEmpty(expectedBody) || actualBody.Equals(expectedBody) || expectedBody.JsonKeyValuesMatch(actualBody);
+        }
+
+        public static bool JsonKeyValuesMatch(this string expectedBody, string actualBody)
+        {
+            JObject expectedJObject;
+            JObject actualJObject;
+
+            try
+            {
+                expectedJObject = JsonConvert.DeserializeObject<JObject>(expectedBody);
+                actualJObject = JsonConvert.DeserializeObject<JObject>(actualBody);
+            }
+            catch (Exception)
+            {
+                return false;
```

Comment:
What happens if this exception isn't caught?  I'm wondering if the users of this test may wish to know that they've got invalid JSON in their expectations and wish to fix it.